src/constants/tooltip.js

```diff
@@ -0,0 +1,81 @@
+export const TOOLTIP_FORMAT_TYPES = {
+  DATE: 'date',
+  DECIMAL: 'decimal',
+  PERCENTAGE: 'percentage'
+};
+
+export const TOOLTIP_FORMATS = {
```

Comment:
, I think google sheet has a reasonable collection of formats for reference

<img width="338" alt="Screen Shot 2020-05-28 at 1 45 41 AM" src="https://user-images.githubusercontent.com/3605556/83119839-0ef12280-a085-11ea-93b4-0cfb55ebcaf1.png">

Reply:
Okay cool, but in next iteration